lib/doc.go

```diff
@@ -0,0 +1 @@
+package lib
```

Comment:
doc.go is an empty file, is it important to keep it right now?

Reply:
right! ack.